src/docs-page.html

```diff
@@ -581,7 +581,7 @@ <h2 class="section-heading">General WebRTC</h2>
 
 
 							<pre><code>
-   const upscaler  = new vectorlyUpscaler(document.getElementById("remoteVideo"), {width: width, height: height});
+   const upscaler  = new vectorlyUpscaler(document.getElementById("remoteVideo"), , {token: 'insert-vectorly-token-here'}, {width: width, height: height});
```

Comment:
As mentioned on slack, {width, height} is not required/supported

Reply:
Is this part of low-level API now?

Reply:
Not sure